build/MUX-Release.yml

```diff
@@ -40,6 +40,20 @@ jobs:
     publishDir : $(Build.ArtifactStagingDirectory)
 
   steps:
+
+  # Download and extract nuget package with non-stubbed MicrosoftTelemetry.h header
+  - task: DownloadPackage@1
+    displayName: 'Download Microsoft.Telemetry.Inbox.Native'
+    inputs:
+      feed: '/3415933f-ac0d-4766-8c0a-3f4c247c25f5'                         # 0
+      view: 'ef61a1c1-003b-4a27-bde5-beec8301021b'                          # Release
+      definition: '2fe60c09-c66f-4275-ae2d-f015c7170c72'                    # Microsoft.Telemetry.Inbox.Native
+      version: '10.0.18362.1-190318-1202.19h1-release.amd64fre'             # latest version
+      downloadPath: '$(System.DefaultWorkingDirectory)'                     # download and extract to repo root
+
+  - script: 'move /Y $(System.DefaultWorkingDirectory)\build\native\inc\MicrosoftTelemetry.h $(System.DefaultWorkingDirectory)\dev\telemetry\'
```

Comment:
This will still succeed even if MicrosoftTelemetry.h is not found at that location. This could happen if the source structure got refactored. This would result in the build succeeding, even though we failed to update the header. We should update this to fail if the header is not found at the expected location.

Reply:
Good point

Reply:
I tested the change by pushing a branch with no MicrosoftTelemetry.h stubbed header checked in, and validated that the build fast-failed: [https://dev.azure.com/microsoft/WinUI/_build/results?buildId=19665375&view=results](https://dev.azure.com/microsoft/WinUI/_build/results?buildId=19665375&view=results)

---
In reply to: [346953912](https://github.com/microsoft/microsoft-ui-xaml/pull/1611#discussion_r346953912) [](ancestors = 346953912)